forte/api/forte_python_module.cc

```diff
@@ -106,7 +106,9 @@ void export_ActiveSpaceSolver(py::module& m) {
              "Set the active space integrals manually")
         .def("compute_fosc_same_orbs", &ActiveSpaceSolver::compute_fosc_same_orbs)
         .def("state_filename_map", &ActiveSpaceSolver::state_filename_map)
-        .def("dump_wave_function", &ActiveSpaceSolver::dump_wave_function);
+        .def("dump_wave_function", &ActiveSpaceSolver::dump_wave_function)
```

Comment:
Add docstrings for all the newly bound methods.

Reply:
Added.